modules/common-attributes.adoc

```diff
@@ -42,8 +42,7 @@ endif::[]
 :ocp: OpenShift Container Platform
 :rhacs-version: 3.68.0
 :ocp-supported-version: 4.5
-:scanner-version: 3.68.0
-:collector-version: 3.68.0-latest
+:collector-version: 3.6.0
```

Comment:
Is it by design that you updated it to 3.`6`.0 here to have a distinction from the current `rhacs-version`?

Reply:
+1. Also, I think this can go, see below

Reply:
I took the content from `COLLECTOR_VERSION` file for the release.
However, there was some confusion that's resolved and `:collector-version:` is gone now. See https://github.com/stackrox/openshift-docs/pull/6#discussion_r793992258